### ISSUE TITLE
The Bound Sword spell and the Sustained Magic Mod dispelling do not work as intended.  

### ISSUE THREAD

**Issue by axfred** [ForgottenGlory/Living-Skyrim#83]
Please delete my previous bug report submitted in error.

Using Living Skyrim 1.4.3
**Describe the bug**
The Bound Sword (BS) spell reserves magicka (Sustained Magic Mod) that does not go away when BS is unequipped or changed to different spell/weapon in hand.  
Using the mod's Dispel Magic spell will not remove the red listing of the Sustained BS spell. 
Recasting the BS Spell and then casting Dispel Magic is the only way I found to remove both red entries in the Magic menu and restore Magicka back to full.

**To Reproduce**
Equip and cast BS spell, then switch to another weapon or spell in same hand.  Check magic menu and red entry for Sustained BS spell still exists.  Cast dispel magic spell using Sustained Magic dispel spell. Recheck magic menu and the red entry is not removed.   Recasting BS spell same hand results in two instances of Sustained BS spells in Magic Menu.  Recasting Dispel magic at this point will remove both instances of Sustained BS and restore magicka to full.

**Expected behavior**
Sustained Bound Sword spell should be removed when unequipped.
Sustained magic dispel spell should remove all Sustained spells and restore magicka to full.

**Screenshots**

**Additional context**

**Comment by axfred** [comment]
This was a previous character save from 1.4.2, and I discovered the Bound Sword was still sustained  after I loaded that save in 1.4.3.  I repeated the reproduce process several times with the same results.  I have not tried on a new character. Will test on new later tonight and I'll post the results.

**Comment by axfred** [comment]
I tried to repeat the process with a new character.  I equipped and summoned bound sword (BS) spell then switched to another weapon in same hand.  Casting the personal Dispel spell in opposite hand to try to remove BS had no effect. When I first started the game I equiped BS in right hand and dispel in left, I cast BS and then dispelled it just fine, the problem if I then switch to anything other than BS in that hand and try to dispel it doesn't work.  I was unable to remove the first instance of sustained BS from the magic menu from that point forward.  That spell sustained uses 50 magicka making it almost useless for a low level spellsword and it can't be removed.  Have not tried this with other bound weapon spells.
